packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -139,6 +140,8 @@ const Dropdown = React.forwardRef(function Dropdown(
   }
 
   const menuItemOptionRefs = useRef(items.map((_) => React.createRef()));
+  const ariaLabelForMenu =
+    ariaLabelMenu || ariaLabel ? `${ariaLabel}__menu` : undefined;
```

Comment:
Instead of setting to undefined, I think we should default it to the label if the ariaLabel prop is not passed